src/TitleDetails/TitleDetailsContainer.js

```diff
@@ -11,30 +11,53 @@ import {
 } from '@folio/stripes-acq-components';
 
 import { PO_LINES_API } from '../common/constants';
-import { titleResource } from '../common/resources';
+import { ITEM_STATUS } from './constants';
+import {
+  checkInResource,
+  locationsResource,
+  orderPiecesResource,
+  pieceResource,
+  titleResource,
+} from '../common/resources';
 import TitleDetails from './TitleDetails';
+import { checkInItems } from './utils';
 
 const TitleDetailsContainer = ({ location, history, mutator, match }) => {
   const titleId = match.params.id;
   const showCallout = useShowCallout();
   const [isLoading, setIsLoading] = useState(false);
   const [title, setTitle] = useState({});
   const [poLine, setPoLine] = useState({});
+  const [pieces, setPieces] = useState([]);
+  const [locations, setLocations] = useState([]);
 
   useEffect(
     () => {
       setIsLoading(true);
       setTitle({});
       setPoLine({});
-      mutator.title.GET()
-        .then(response => {
-          setTitle(response);
+      setPieces([]);
+
+      Promise.all([mutator.title.GET(), mutator.locations.GET()])
+        .then(responses => {
```

Comment:
please use destructuring assignment with proper var names instead of `response0, response2